src/main/java/com/redislabs/redisai/RedisAI.java

```diff
@@ -16,6 +17,8 @@
 import redis.clients.jedis.util.Pool;
 import redis.clients.jedis.util.SafeEncoder;
 
+import javax.xml.crypto.Data;
```

Comment:
@filipecosta90 is it needed?

Reply:
no. probably imported wrongfully while testing approaches. changing